tests/Unit/UseCases/AddSubscription/SubscriptionRequestTest.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+
+namespace WMDE\Fundraising\Frontend\Tests\Unit\UseCases\AddSubscription;
+
+use WMDE\Fundraising\Frontend\UseCases\AddSubscription\SubscriptionRequest;
+
+/**
+ * @covers WMDE\Fundraising\Frontend\UseCases\AddSubscription\SubscriptionRequest
+ *
+ * @licence GNU GPL v2+
+ * @author Gabriel Birke < gabriel.birke@wikimedia.de >
+ */
+class SubscriptionRequestTest extends \PHPUnit_Framework_TestCase {
+	public function testSubscriptionRequestCanBeInitializedFromArray() {
+		$subscriptionRequest = SubscriptionRequest::createFromArray( [
+			'firstName' => 'Nyan',
+			'lastName' => 'Cat',
+			'salutation' => 'Herr',
+			'title' => 'Prof. Dr. Dr.',
```

Comment:
Took me till seeing this to understand what title meant. In the request TDO this was certainly not clear to me. This is another reason to not use the dynamic mapping magic: it forces the setter names to match those of the presentation interface.